src/TicTacToeModel.java

```diff
@@ -0,0 +1,77 @@
+public class TicTacToeModel {
+    public enum Player {X, O, EMPTY}
+    private final Player[][] board = new Player[3][3];
+    private Player currentPlayer;
+
```

Comment:
Consider adding comments to explain the purpose of each method and variable for better readability. This will help others understand the code more easily.